.circleci/config.yml

```diff
@@ -434,6 +435,18 @@ commands:
           path: fastlane/test_output/xctest
           destination: scan-test-output
 
+  checkout-with-minimal-permissions:
+    steps:
+      - revenuecat/remove-all-ssh-keys
+      - revenuecat/checkout-with-key:
+          fingerprint: "SHA256:TY+ziSK99HtCkgT7rILg/o2G05KSV8BxmQht0ApvYng"
+      - revenuecat/assert-git-clone-fails:
+          repository: paywall-preview-resources
+      - revenuecat/assert-git-clone-fails:
+          repository: paywall-rendering-validation
+      - revenuecat/assert-git-clone-fails:
+          repository: khepri
```

Comment:
Is this one needed because it requires different permissions or is it just to be extra-sure?
I'm a bit scared of exposing this name in the wild 👀